### PR TITLE
Tap feedback on perils

### PR DESCRIPTION
**Pull request by gustafguner**
Added haptic tap feedback on perils

### CODE REVIEW

File: Src/Screens/DashboardScreen/Viewables/ExpandableRow.swift [hunk 1]
```diff
@@ -56,10 +56,19 @@ extension ExpandableRow: Viewable {
...
 
         let expandableStackView = UIStackView()
+        expandableStackView.backgroundColor = .white
         expandableStackView.axis = .vertical
 
-        bag += expandableStackView.addArranged(content)
-
+        let contentWrapper = UIControl()
+        
+        bag += contentWrapper.add(content) { buttonView in
```
Comment (Iamsamwhoami) [inline Src/Screens/DashboardScreen/Viewables/ExpandableRow.swift]: `contentWrapperView` maybe instead of `buttonView`

Comment (gustafguner) [reply Src/Screens/DashboardScreen/Viewables/ExpandableRow.swift]: yeah that's more consistent 👍

File: Src/Screens/DashboardScreen/Viewables/Peril.swift [hunk 2]
```diff
@@ -99,14 +99,27 @@ extension Peril: Reusable {
...
             }
             
-            bag += perilView.signal(for: .touchUpInside).animated(style: SpringAnimationStyle.lightBounce()) { _ in
-                perilView.transform = CGAffineTransform.identity
+            bag += merge(
+                perilView.signal(for: .touchUpInside),
+                perilView.signal(for: .touchUpOutside),
+                perilView.signal(for: .touchCancel)
+                ).withLatestFrom(touchDownDateSignal.atOnce().plain())
+                .delay(by: { _, date in date.timeIntervalSinceNow < -0.2 ? 0 : 0.2 })
```
Comment (Iamsamwhoami) [inline Src/Screens/DashboardScreen/Viewables/Peril.swift]: maybe we should put this in an extension on `SignalProvider`, something like `delayedTouchDown` 🤔

Comment (gustafguner) [reply Src/Screens/DashboardScreen/Viewables/Peril.swift]: hmm yeah that's true, definitely not the last time we're gonna need this 👌

Comment (gustafguner) [reply Src/Screens/DashboardScreen/Viewables/Peril.swift]: but rather `delayedTouchCancel` I guess 🤔

Comment (Iamsamwhoami) [reply Src/Screens/DashboardScreen/Viewables/Peril.swift]: Oh, yes! `delayedTouchCancel` is a good name!

File: Src/Generic/Extensions/UIControl+DelayedTouchCancel.swift [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+//
+//  Created by Gustaf Gunér on 2019-05-29.
+//
+
+import Flow
+import Foundation
+import UIKit
+
+extension UIControl {
+    func delayedTouchCancel(delay: Double = 0.2) -> Signal<Bool> {
```
Comment (Iamsamwhoami) [inline Src/Generic/Extensions/UIControl+DelayedTouchCancel.swift]: This could be a `Signal<Void>` right? or does the Bool mean something?

Comment (Iamsamwhoami) [inline Src/Generic/Extensions/UIControl+DelayedTouchCancel.swift]: Usually time is presented with the TimeInterval type, which is a typealias for a Double, but still 😄

Comment (gustafguner) [reply Src/Generic/Extensions/UIControl+DelayedTouchCancel.swift]: Yep yep, you're right

Comment (gustafguner) [reply Src/Generic/Extensions/UIControl+DelayedTouchCancel.swift]: The more descriptive the better 😄

**Pull request merged**